src/test/resources/config/spark/w_all_tpcds-delta.yaml

```diff
@@ -27,6 +27,13 @@ phases:
   sessions:
   - tasks:
     - template_id: data_maintenance_delta
+- id: data_maintenance_dependent
```

Comment:
Could you add a similar one to `w_all_tpcds_single_session-*` too? Those ones are executed through the Session connector, it would be great to have it running in the CI through that connection too.

Reply:
Added three new tests with a single session and the dependent task added to the task list. Since dependent tasks are not currently supported via a SparkConnection, I had to separate the configurations. Is this what you had in mind?

Reply:
I had forgotten it was not supported in `SparkSession`. This looks fine, though it increased the testing time for Spark JDBC CI from ~15mins to ~25mins; this seems still reasonable but let's keep it in mind moving forward.
I'm not sure if there is a GH issue to include dependent task execution in `SparkSession`; if there is not, please create one so we can tackle this at some point. Thanks

Reply:
GH issue: [120](https://github.com/microsoft/lst-bench/issues/120)